docs/1.0. Certificate Provisioning.md

```diff
@@ -206,13 +206,18 @@ An EST Client MUST provide a method to replace the manufacturer-issued TLS Clien
 
 An EST Client MAY ignore all timestamps in the certificate validity periods during the certificate provisioning process if it does not accurately know the current time, see [Bootstrapping Remote Secure Key Infrastructures - Section 2.6.1](https://tools.ietf.org/html/draft-ietf-anima-bootstrapping-keyinfra-41#section-2.6.1) for more information.
 
-If an EST Client implements the NMOS IS-04 Node API, then the EST Client SHOULD advertise that it supports BCP-003-03 using the 'caps' attribute of the Node API. Control interfaces can identify all Nodes which support BCP-003-03 by a key 'cert' and Uniform Resource Name (URN) having the base 'urn:x-nmos:control:cert' followed by a '/' character and the supported version of this BCP.
+If an EST Client implements the NMOS IS-04 Node API, then the EST Client SHOULD advertise that it supports BCP-003-03 using the 'tags' attribute of the Node API's `/self` endpoint. Control interfaces can identify all Nodes which support BCP-003-03 by a key 'cert'. The supported version of this BCP is defined by the 'versions' list and if EST is currently active by the 'enabled' boolean.
 
-**Example:** The 'caps' attribute of a NMOS Node
-```
+**Example:** The 'tags' attribute of a NMOS Node
+```json
 ...
-"caps": {
-  "cert": "urn:x-nmos:control:cert/v1.0",
+"tags": {
+  "cert": {
```

Comment:
```suggestion
  "urn:x-nmos:tag:cert": {
```

Matching the precedent in https://github.com/AMWA-TV/nmos-parameter-registers/blob/master/tags/grouphint.md a little more closely. The version could potentially make its way into the tag too.